File: 601_700/657_robot_return_to_origin/Solution.py
class Solution:
    def judgeCircle(self, moves: str) -> bool:
        x, y= 0,0
        for i in moves:
            if i == 'U':
                x += 1
            if i == 'D':
                x -= 1
            if i == 'L':
                y += 1
            if i == 'R':
                y -= 1
        return x == 0 and y == 0

File: 601_700/657_robot_return_to_origin/Solution.rs
impl Solution {
    pub fn judge_circle(moves: String) -> bool {
        let (mut x, mut y) = (0,0);
        for i in moves.bytes() {
            let c = i as char;
            match c {
                'U' => x+=1,
                'D' => x-=1,
                'R' => y+=1,
                'L' => y-=1,
                _ => (),
            }
        }
        x == 0 && y == 0
    }
}
